lib/checks/aria/aria-prohibited-attr-evaluate.js

```diff
@@ -27,12 +27,9 @@ import standards from '../../standards';
  */
 function ariaProhibitedAttrEvaluate(node, options, virtualNode) {
   const prohibited = [];
-  const role = getRole(virtualNode);
-
-  if (!role) {
-    return false;
-  }
 
+  // From Scott O'Hara on asking about HTML elements with no role mapping: no mapping = generic
+  const role = getRole(virtualNode) || 'generic';
```

Comment:
I'm not so sure I agree with that suggestion of Scott, nor with the implication of it. I don't think we should call every use of aria-label on an element we don't have a role for as a failure of SC 4.1.2. That seems a bit of a stretch to me. A best practice might be more appropriate. I'm going to get some more Deque folks to weigh in on that one first.